requirements.txt

```diff
@@ -17,15 +17,18 @@ django-phone-field==1.8.0
 django-storages==1.9.1
 docopt==0.6.2
 docutils==0.15.2
+googlemaps==4.2.0
 gunicorn==20.0.4
 idna==2.9
 jmespath==0.10.0
 oauthlib==3.1.0
 packaging==20.1
 Pillow==7.0.0
 pip-upgrader==1.4.15
+pkg-resources==0.0.0
```

Comment:
remove pkg-resources=0.0.0. from requirements haikuagi available kwa server

Reply:
Done

Removed pkg-resources and google-maps from requirements